src/fidesops/schemas/user.py

```diff
@@ -1,108 +1,8 @@
-import re
-from datetime import datetime
-from typing import Optional
-
-from pydantic import validator
-
-from fidesops.schemas.base_class import BaseSchema
-from fidesops.schemas.oauth import AccessToken
-from fidesops.util.cryptographic_util import b64_str_to_str
-
-
-class UserUpdate(BaseSchema):
-    """Data required to update a FidesopsUser"""
-
-    first_name: Optional[str]
-    last_name: Optional[str]
-
-
-class UserCreate(UserUpdate):
-    """Data required to create a FidesopsUser"""
-
-    username: str
-    password: str
-
-    @validator("username")
-    def validate_username(cls, username: str) -> str:
-        """Ensure password does not have spaces"""
-        if not username:
-            raise ValueError("Must enter username.")
-        if " " in username:
-            raise ValueError("Usernames cannot have spaces.")
-        return username
-
-    @validator("password")
-    def validate_password(cls, password: str) -> str:
-        """Add some password requirements"""
-        decoded_password = b64_str_to_str(password)
-        if len(decoded_password) < 8:
-            raise ValueError("Password must have at least eight characters.")
-        if re.search("[0-9]", decoded_password) is None:
-            raise ValueError("Password must have at least one number.")
-        if re.search("[A-Z]", decoded_password) is None:
-            raise ValueError("Password must have at least one capital letter.")
-        if re.search("[a-z]", decoded_password) is None:
-            raise ValueError("Password must have at least one lowercase letter.")
-        if re.search(r"[\W]", decoded_password) is None:
-            raise ValueError("Password must have at least one symbol.")
-
-        return decoded_password
-
-
-class UserLogin(BaseSchema):
-    """Similar to UserCreate except we do not need the extra validation on username and password"""
-
-    username: str
-    password: str
-
-    @validator("password")
-    def validate_password(cls, password: str) -> str:
-        """Convert b64 encoded password to normal string"""
-        return b64_str_to_str(password)
+from fideslib.schemas.base_class import BaseSchema
 
 
 class UserPasswordReset(BaseSchema):
```

Comment:
I think it would make sense. The reason it wasn't done now is because we decided more discussion was needed around how to handle password resets, and we were therefore leaving it out for now.

That said if it makes sense to go ahead and move the schema now I can do that.

Reply:
we can follow-up. Any other schemas that need to be moved? Do we have a ticket yet?